docs/contributor/01-00-testing.md

```diff
@@ -0,0 +1,31 @@
+# Testing
+
+The Eventing Publisher Proxy uses the following testing activities:
+
+## Unit Tests and Env-Tests
+
+To run the unit and integration tests, you must run the following command:
```

Comment:
for better consistency, we should pick one word - either "env-tests" (like in the headline) or "integration tests" (like in the text).

Reply:
Done